### ISSUE TITLE
Files | Software not found

### ISSUE THREAD

**Issue by Manish-757** [ChrisTitusTech/winutil#1315]
**Describe the bug**
Files software is not getting installed, it seems like the listed package is not there in winget anymore

**To Reproduce**
1. Select Files File Explorer
2. Click on Install
3. See error

**Expected behavior**
Software should be installed

**Screenshots**
![image](https://github.com/ChrisTitusTech/winutil/assets/154927739/6337da1b-e898-4f78-833a-b34add74aea7)

**Comment by jstack01** [comment]
So it appears that files community is no longer available on winget. Per applications.json, the winget package id listed under winutil is "YairAichenbaum.Files", however this yields no results using winget search. Furthermore, according to https://winget.run/pkg/FilesCommunity/Files , the winget package id should be "FilesCommunity.Files" however this also returns no results. So either the package was removed from winget entirely, or it is under a new winget package id which I am unsure of. I am not sure how to install packages using choco with winutil, however you can run "choco install files" using powershell and the install should work just fine. A commit is needed to remove/update the winget package id for "Files File Explorer".

**Comment by munkk01** [comment]
- Main issue discussing the removal of the app from Winget
https://github.com/files-community/Files/issues/8653

- ID of the application discoverable via WinGet
https://github.com/files-community/Files/issues/8653#issuecomment-1066132380

- Potential issue(s) downloading it this way
https://github.com/files-community/Files/issues/8653#issuecomment-1080747580

Attempted an install on a 23H2 machine resulted in:
```powershell
winget install 9NGHP3DX8HDX
```
->
`Failed to install or upgrade Microsoft Store package. Error code: 0x803f8001`

Full command output:
```
winget install 9NGHP3DX8HDX --force

Found Files App [9NGHP3DX8HDX] Version Unknown
This package is provided through Microsoft Store. winget may need to acquire the package from Microsoft Store on behalf of the current user.
Agreements for Files App [9NGHP3DX8HDX] Version Unknown
Version: Unknown
Publisher: Yair A
Publisher Url: https://files.community
Description:
  Introducing Files, the ultimate file manager for Windows. With its sleek and intuitive design, navigating through your files and documents has never been easier. Files features tabs for easy switching between different folders, a column layout for quick file browsing, and dual pane support for efficient file management. In addition, you can easily create and extract zipped archives with just a few clicks, making file compression and decompression a breeze.

  Files also offers advanced features such as file tagging for easy organization, support for QuickLook for previewing files without opening them, and the ability to customize the background color to match your personal style. Whether you're a power user or just looking for an alternative to File Explorer, Files has everything you need to keep your files organized and easily accessible. With its combination of powerful features and ease of use, Files is the ultimate file management solution for Windows.

  Purchasing Files through the Microsoft Store will help support the developers and enable them to continue to develop and improve the app with new features and support for future updates.
License: ms-windows-store://pdp/?ProductId=9NGHP3DX8HDX
Privacy Url: https://github.com/files-community/Files/blob/HEAD/.github/PRIVACY.md
Copyright: Files Community
Agreements:
Category: Productivity
Free Trial: No
Terms of Transaction: https://aka.ms/microsoft-store-terms-of-transaction
Seizure Warning: https://aka.ms/microsoft-store-seizure-warning
Store License Terms: https://aka.ms/microsoft-store-license


The publisher requires that you view the above information and accept the agreements before installing.
Do you agree to the terms?
[Y] Yes  [N] No: y
Starting package install...
Failed to install or upgrade Microsoft Store package. Error code: 0x803f8001
```

**Comment by jstack01** [comment]
Well... I got the very same error message as you did when running "winget install 9NGHP3DX8HDX" . I went onto the Microsoft Store and attempted to download the same application, however I ended up recieving the same error. This was on Windows 10 22H2 (signed out of Microsoft). I then went to my Windows 11 22H2 machine and discovered the "Files App" is $8.99, which can probably explain why I couldn't download the package on my Windows 10 computer using winget or the MStore. 
![app](https://github.com/ChrisTitusTech/winutil/assets/45379549/aa619565-5071-4d41-98fd-f920142a25b7)
